activity-be/inoculation.sql

```diff
@@ -0,0 +1,17 @@
+create table inoculation
+(
+	id serial not null
+		constraint inoculation_pkey
+			primary key,
+	drug varchar(50) not null,
+	dose varchar(50) not null,
```

Comment:
when saving inoculation by batch, dose is irrelevant since it depends on weight. Should be nullable